src/nxdoc/nuxeo-server/administration/configuration-parameters-index-nuxeoconf.md

```diff
@@ -2276,17 +2276,45 @@ For RestClient protocol a comma separated list of URL. If empty an in JVM embedd
 </tr>
 <tr>
 <td colspan="1">`elasticsearch.restClient.connectionTimeoutMs`</td>
-<td colspan="1">A timeout in milliseconds until a connection is established (RestClient)</td>
+<td colspan="1">A timeout in milliseconds until a connection is established</td>
 <td colspan="1">`5000`</td>
 <td colspan="1">Since 9.10</td>
 </tr>
 <tr>
 <td colspan="1">`elasticsearch.restClient.socketTimeoutMs`</td>
-<td colspan="1">A maximum period, in milliseconds, of inactivity between two consecutive data packets (RestClient)</td>
+<td colspan="1">A maximum period, in milliseconds, of inactivity between two consecutive data packets</td>
 <td colspan="1">`20000`</td>
 <td colspan="1">Since 9.10</td>
 </tr>
-
+<tr>
+<td colspan="1">`elasticsearch.restClient.username`</td>
+<td colspan="1">A username for client basic authorization (X-Pack)</td>
```

Comment:
"Basic Authentication" instead "basic authorization" also it sounds like it is only for X-Pack which is not true.